framework/include/materials/ADMaterial.h

```diff
@@ -17,7 +17,19 @@
   using ADMaterial<compute_stage>::_qp;                                                            \
   using ADMaterial<compute_stage>::_ad_grad_zero;                                                  \
   using ADMaterial<compute_stage>::_ad_zero;                                                       \
-  using ADMaterial<compute_stage>::coupledComponents
+  using ADMaterial<compute_stage>::_qrule;                                                         \
+  using ADMaterial<compute_stage>::_JxW;                                                           \
+  using ADMaterial<compute_stage>::_coord;                                                         \
+  using ADMaterial<compute_stage>::_q_point;                                                       \
+  using ADMaterial<compute_stage>::_assembly;                                                      \
+  using ADMaterial<compute_stage>::_mesh;                                                          \
+  using ADMaterial<compute_stage>::coupled;                                                        \
+  using ADMaterial<compute_stage>::coupledComponents;                                              \
+  using ADMaterial<compute_stage>::isParamValid;                                                   \
+  using ADMaterial<compute_stage>::getParam;                                                       \
```

Comment:
This won't work. See discussion in #12657

Reply:
I'm not even using it, so I'll drop it from this PR